### PR TITLE
Add a quick menu on desktop

### PR DESCRIPTION
**Pull request by Flaburgan**
This pull request adds a menu on the left of the site to be able to navigate between the pages. There is actually two breakpoints, the menu seats next to the tiles on a big screen, switch at the top of the screen on screen < 1366 and is not displayed at all on mobile (< 642px). I will add a drawer menu on mobile on a following pull request (or in this one if it's not merged before).

### CODE REVIEW

**Comment by Flaburgan** [comment]
Ready to be reviewed, I included the switch on mobile.

File: src/views/base.njk [hunk 1]
```diff
@@ -1,23 +1,35 @@
...
+  </head>
+  <body>
+    <a id="drawer-icon" href="#drawer-opened"></a>
+    <nav id="menu" class="tile">
+      <a class="close-drawer" href="#">x</a>
+      <ul>
+        <li><a href="/">Home</a></li>
+        <li><a href="/global-stats">Global stats</a></li>
+        <li><a href="/diaspora">diaspora*</a></li>
+        <li><a href="/hubzilla">Hubzilla</a></li>
```
Comment (jaywink) [inline src/views/base.njk]: Missing `/friendica`.

**Comment by Flaburgan** [comment]
Oops yes I removed both when removing redmatrix. Corrected.

**Comment by jaywink** [comment]
Conflicts due to previous merge :P

**Comment by jaywink** [comment]
Awesome, huge thanks!

**Pull request merged**